apps/music/js/ui/views/list_view.js

```diff
@@ -95,6 +95,8 @@ var ListView = {
       headerLi = document.createElement('li');
       headerLi.className = 'list-header';
       headerLi.textContent = this.lastFirstLetter || '?';
+      headerLi.setAttribute('role', 'heading');
+      headerLi.setAttribute('aria-level', '2');
```

Comment:
@eeejay When music app is in searching modes, like searching in tiles or lists, the search results will have something like the list headers(Artists, Albums or Tracks), do we need to also set role and aria-level for them?